solidity/contracts/vault/TBTCVault.sol

```diff
@@ -249,6 +223,64 @@ contract TBTCVault is IVault, Governable {
         upgradeInitiatedTimestamp = 0;
     }
 
+    /// @notice Allows the governance of the TBTCVault to recover any ERC20
+    ///         token sent mistakenly to the TBTC token contract address.
+    /// @param token Address of the recovered ERC20 token contract.
+    /// @param recipient Address the recovered token should be sent to.
```

Comment:
`@param`'s should not end with a dot.

Reply:
It should now 😆 We changed it in https://github.com/keep-network/tbtc-v2/pull/281:

> Each `@param` ends with a dot `.` now. It is a different approach from the one we
used in `@keep-core/ecdsa` and `@keep-core/random-beacon` but it is time to
acknowledge `@param` descriptions in `tbtc-v2` are much longer and they often do
form (multiple) sentences.